### DISCUSSION TITLE
Archiving - Mixed plot sizes to fill drive

### DISCUSSION THREAD

**Discussion by FVierra27** [ericaltendorf/plotman#887] [Ideas]
Is it possible, when archiving, to check a destination for amount of k32, k33, and k34 files then choose the first disk available with less then the amount of plots you want to go on that drive?

For example an 18TB drive will only fill 132 K32's on one drive. So later can come back and fill with K33's to get (nearly) 100% fill?

**Comment by altendky** [comment]
No, that feature doesn't exist.  I personally am not convinced that non-homogeneity is worth the absolute max of 0.6% improved space usage.  In practice on my 18TB drives it's more like 0.25%.  I won't say that I wouldn't merge a PR for this, but I am unlikely to get to developing it myself given the other tasks.  Sorry.  `:[`

**Reply by FVierra27** [reply]
No worries, just thought it might be a good idea.

**Reply by altendky** [reply]
Yeah, it's certainly one of the things people do.  I could imagine it taking a form that could fit with plotman but I think that would take some work.
